logfire/_internal/integrations/system_metrics.py

```diff
@@ -0,0 +1,189 @@
+from __future__ import annotations
+
+import contextlib
+import sys
+from typing import TYPE_CHECKING, Dict, Iterable, Literal, Optional, cast
+
+from opentelemetry.metrics import CallbackOptions, Observation
+
+if TYPE_CHECKING:
+    from typing_extensions import LiteralString
+
+    from logfire import Logfire
+
+try:
+    import psutil
+    from opentelemetry.instrumentation.system_metrics import (
+        _DEFAULT_CONFIG,  # type: ignore
+        SystemMetricsInstrumentor,
+    )
+except ModuleNotFoundError as e:  # pragma: no cover
+    raise RuntimeError(
```

Comment:
```suggestion
    raise ImportError(
```
?

Reply:
This is just following an existing pattern in the codebase:

https://github.com/pydantic/logfire/blob/fe127144cfe89e221e92268317383b1cfa7cfd79/logfire/_internal/integrations/fastapi.py#L27

https://github.com/pydantic/logfire/blob/fe127144cfe89e221e92268317383b1cfa7cfd79/logfire/_internal/integrations/django.py#L8

@Kludex any reason you chose `RuntimeError` for fastapi?